README.md

```diff
@@ -397,7 +397,24 @@ ANSWERS.addComponent('SearchBar', {
   // Note that WCAG compliance is not guaranteed if a form is not used as the context.
   useForm: 'true',
   // Optional, the input element used for searching and wires up the keyboard interaction
-  inputEl: '.js-yext-query'
+  inputEl: '.js-yext-query',  
+  // Optional, options to pass to the geolocation api.
+  // https://developer.mozilla.org/en-US/docs/Web/API/PositionOptions
+  geolocationOptions: {
+    // Optional, whether to improve accuracy at the cost of response time and/or power consumption, defaults to false.
+    enableHighAccuracy: false,
+    // Optional, the maximum amount of time (in ms) a geolocation call is allowed to take before defaulting, defaults to 1000ms.
+    timeout: 1000,
+    // Optional, the maximum amount of time (in ms) to cache a geolocation call, defaults to 300000ms (5 minutes).
+    maximumAge: 300000,
+  },
+  // Optional, options for the alert given when geolocation defaults.
```

Comment:
options for an alert when the geolocation call fails?

Reply:
I wanted to avoid using the word "fail" but this might be ptsd from 6th grade